samples/04_gis_analysts_data_scientists/forecasting_monthly_rainfall_in_California_using_deeplearning_timeseries_model_from_arcgis_learn.ipynb

```diff
@@ -0,0 +1,2195 @@
+{
```

Comment:
<p>Forecasting requires the format of the date column to be in datetime. If the date column is not in the datetime format, it can be changed to datetime by using the pd.to_datetime() method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>